pkg/test/resourcefixture/testdata/basic/pubsublite/v1alpha1/pubsublitesubscription/pubsublitesubscriptionbasic/create.yaml

```diff
@@ -18,8 +18,10 @@ metadata:
   name: pubsublitesubscription-${uniqueId}
 spec:
   deliveryConfig:
-  - deliveryRequirement: DELIVER_AFTER_STORED
```

Comment:
hm, did this not fail before?

Reply:
Great question ... @gemmahou is this on our e2e skiplist?

Reply:
I believe so. I skipped alpha resources.

Reply:
Ah!  That'll do it :-)